src/main/java/ted/task/TaskList.java

```diff
@@ -0,0 +1,77 @@
+package ted.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    /**
+     * Internal stores of tasks
+     */
+    private ArrayList<Task> tasks;
+
+    /**
+     * Construct a Task list instance
+     * @param tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Add a task into tasks list
+     * @param task
+     */
+    public void add(Task task) {
+        this.tasks.add(task);
+    }
+
+    /**
+     * Get a task from tasks list
+     * @param index
+     * @return
```

Comment:
I think you forgot to complete some of the javadocs in this class

Reply:
Good catch!! Will fix it!